bin/sqs_cli.py

```diff
@@ -0,0 +1,155 @@
+
+
+def get_client():
+    session = boto3.session.Session(
+        aws_access_key_id=os.environ.get("CRASHPUBLISH_ACCESS_KEY"),
+        aws_secret_access_key=os.environ.get("CRASHPUBLISH_SECRET_ACCESS_KEY"),
+    )
+    client = session.client(
+        service_name="sqs",
+        region_name=os.environ.get("CRASHPUBLISH_REGION"),
+        endpoint_url=os.environ.get("CRASHPUBLISH_ENDPOINT_URL"),
+    )
+    return client
+
+
+@click.group()
+def sqs_group():
+    """Local dev environment SQS manipulation script."""
+
+
+@sqs_group.command("list_messages")
+@click.argument("queue")
+@click.pass_context
+def list_messages(ctx, queue):
+    """List messages in queue."""
+    conn = get_client()
+    try:
+        resp = conn.get_queue_url(QueueName=queue)
+        queue_url = resp["QueueUrl"]
+    except conn.exceptions.QueueDoesNotExist:
+        click.echo("Queue %s does not exist.")
+        return
+
+    # NOTE(willkg): Since the VisibilityTimeout is set to VISIBILITY_TIMEOUT and
+    # messages aren't deleted, items aren't pulled out of the queue permanently.
+    # However, if you run list_messages twice in rapid succession, VisibilityTimeout may
+    # not have passed, so we wait the timeout amount first.
+    time.sleep(VISIBILITY_TIMEOUT)
+
+    is_empty = True
+    while True:
+        resp = conn.receive_message(
+            QueueUrl=queue_url, WaitTimeSeconds=0, VisibilityTimeout=VISIBILITY_TIMEOUT,
+        )
+        msgs = resp.get("Messages", [])
+        if not msgs:
+            break
+
+        is_empty = False
+        for msg in msgs:
+            click.echo("%s" % msg["Body"])
+
+    if is_empty:
+        click.echo("Queue %s is empty." % queue)
+
+
+@sqs_group.command("send_message")
+@click.argument("queue")
+@click.argument("message")
+@click.pass_context
+def send_message(ctx, queue, message):
+    """Add a message to a queue."""
+    conn = get_client()
+    try:
+        resp = conn.get_queue_url(QueueName=queue)
+        queue_url = resp["QueueUrl"]
+    except conn.exceptions.QueueDoesNotExist:
+        click.echo("Queue %s does not exist.")
+        return
+
+    conn.send_message(QueueUrl=queue_url, MessageBody=message)
+    click.echo("Message sent.")
+
+
+@sqs_group.command("list_queues")
+@click.pass_context
+def list_queues(ctx):
+    """List queues."""
+    conn = get_client()
+    resp = conn.list_queues()
+
+    for queue_url in resp.get("QueueUrls", []):
+        queue_name = queue_url.rsplit("/", 1)[1]
+        click.echo(queue_name)
+
+
+@sqs_group.command("create")
+@click.argument("queue")
+@click.pass_context
+def create(ctx, queue):
+    """Create SQS queue."""
+    conn = get_client()
+    validate_queue_name(queue)
+    try:
+        conn.get_queue_url(QueueName=queue)
+        click.echo("Queue %s already exists." % queue)
+        return
+    except conn.exceptions.QueueDoesNotExist:
+        pass
+    conn.create_queue(QueueName=queue)
```

Comment:
This creates a standard queue. I'm pretty sure that's what we're going to use. If we decide to use a FIFO queue, then we'll update this then.